woocommerce/woocommerce
# Reintroduce JITMs in the WC Admin.

### All Submissions:






### Changes proposed in this Pull Request:


Enable the JITMs in the Admin sections. 

![screenshot-fermarichal jurassic tube-2022 09 09-17_17_47](https://user-images.githubusercontent.com/1314156/189438616-0b8613a6-e06c-4f13-9447-e0760a76223a.png)

![screenshot-fermarichal jurassic tube-2022 09 09-17_14_11](https://user-images.githubusercontent.com/1314156/189438641-e346a87c-7b6c-4bb2-be60-fde7b61d10cd.png)

![screenshot-fermarichal jurassic tube-2022 09 09-17_36_36](https://user-images.githubusercontent.com/1314156/189438972-b91a4c58-5dc4-410b-9d52-2df697dc20dc.png)


The JITMs haven't been added to the Onboarding wizard in order to keep the same experience. We can add them later if we want.

Closes # 60-gh-woocommerce/mothra-private

### How to test the changes in this Pull Request:

1. Install WC on a connected Jetpack site.
2. To be able to see a testing JITM, you'll need to add `&test_jitm=/1/` at the end of a WooCommerce URL. It will work on every WooCommerce page except for: `Products > Categories`, `Products > Tags`, `Products > Attributes`, and `Products > Reviews`. 
3. Verify that the JITMs look good using the new navigation.
4. They should look good also with different browser widths.
5. Verify that they are not being shown in the Onboarding wizard.

### Other information:




### FOR PR REVIEWER ONLY:
